src/track/taglib/trackmetadata_common.h

```diff
@@ -71,8 +71,11 @@ inline TagLib::String uuidToTString(
 
 inline QString formatBpm(
         const TrackMetadata& trackMetadata) {
-    return Bpm::valueToString(
-            trackMetadata.getTrackInfo().getBpm().value());
+    const Bpm bpm = trackMetadata.getTrackInfo().getBpm();
+    if (!bpm.isValid()) {
+        return {};
+    }
+    return Bpm::valueToString(bpm.value());
```

Comment:
Should we make `valueToString` a member of the Bpm class (`bpm.toString()`) and make the check internally?

Reply:
Probably, because now the check is done twice.

Reply:
Let's do this separately. I didn't check what else is affected and if it applies to other values.